### PR TITLE
Add TypeScript to Backend

### PR DESCRIPTION
**Pull request by 13bfrancis**
This will add TypeScript to API

### CODE REVIEW

**Review by abelzer212 — APPROVED** [review]

File: services/app-api/handlers/measures/create.ts [hunk 1]
```diff
@@ -3,21 +3,32 @@ import dynamoDb from "../../libs/dynamodb-lib";
 import { createCompoundKey } from "../dynamoUtils/createCompoundKey";
 
 export const createMeasure = handler(async (event, context) => {
+  if (!event.body) return; //throw
+  const body = JSON.parse(event.body);
+  if (!event.pathParameters) return; // throw error message
```
Comment (jdinh8124) [inline services/app-api/handlers/measures/create.ts:8]: The error messages going to be handled in the api branch?

File: services/app-api/handlers/measures/get.ts [hunk 2]
```diff
@@ -4,14 +4,23 @@ import { convertToDynamoExpression } from "../dynamoUtils/convertToDynamoExpress
...
+    !event.pathParameters.coreSet
+  )
+    return; // throw error message
+
   const state = event.pathParameters.state;
   const year = event.pathParameters.year;
   const coreSet = event.pathParameters.coreSet;
 
+  console.log(state, year, coreSet);
   const params = {
```
Comment (jdinh8124) [inline services/app-api/handlers/measures/get.ts:20]: console log from our debugging earlier, not sure if you want that still

File: services/app-api/libs/debug-lib.ts [hunk 3]
```diff
@@ -1,3 +1,5 @@
+//@ts-nocheck
+
 import util from "util";
 import AWS from "aws-sdk";
```
Comment (jdinh8124) [inline services/app-api/libs/debug-lib.ts:5]: wondering if we could blow this file up, considering it seemed like boilerplate

**Review by jdinh8124 — APPROVED** [review]
Thanks for getting this done! Going to help us a lot down the road

File: services/app-api/handlers/measures/get.ts (hunk 2 above)
Comment (13bfrancis) [reply services/app-api/handlers/measures/get.ts:20]: I'll get that out. Good catch

File: services/app-api/libs/debug-lib.ts (hunk 3 above)
Comment (13bfrancis) [reply services/app-api/libs/debug-lib.ts:5]: Another good catch. I'll see if it's used

File: services/app-api/handlers/coreSets/create.ts [hunk 4]
```diff
@@ -37,13 +46,20 @@ export const createCoreSet = handler(async (event, context) => {
...
 });
 
-const createDependentMeasures = async (state, year, coreSet, type) => {
+const createDependentMeasures = async (
+  state: string,
+  year: number,
+  coreSet: string,
+  type: string
+) => {
+  if (year !== 2021 && year !== 2022) return;
```
Comment (benjaminpaige) [inline services/app-api/handlers/coreSets/create.ts:61]: Can we use the keys from the static measuresList for this?

Comment (13bfrancis) [reply services/app-api/handlers/coreSets/create.ts:61]: Oooh. Really good idea. We should definitely be able to do that. That will be much more dynamic too

Comment (abelzer212) [reply services/app-api/handlers/coreSets/create.ts:61]: No we can't the Keys are on the frontend, and this lives in AWS

File: services/app-api/handlers/measures/create.ts (hunk 1 above)
Comment (13bfrancis) [reply services/app-api/handlers/measures/create.ts:8]: We'll handle that afterwards

File: services/app-api/libs/debug-lib.ts (hunk 3 above)
Comment (13bfrancis) [reply services/app-api/libs/debug-lib.ts:5]: I'm gonna leave it for now

**Review by benjaminpaige — APPROVED** [review]

**Pull request merged**